pkg/apis/eventsource/v1alpha1/types.go

```diff
@@ -421,6 +421,9 @@ type SNSEventSource struct {
 	// Metadata holds the user defined metadata which will passed along the event payload.
 	// +optional
 	Metadata map[string]string `json:"metadata,omitempty" protobuf:"bytes,7,rep,name=metadata"`
+	// ValidateSignature is boolean that can be set to true for SNS signature verification
+	// +optional
+	ValidateSignature bool `json:"validateSignature,omitempty" protobuf:"varint,8,...."`
```

Comment:
What I meant was changing `bytes` to `varint`.

Reply:
Oh right I am sorry. made the change now. Actually I don't know how protobuf works, tried following the other variables in the struct and somehow missed checking what bool variables should have, but will definitely put some time to understand it

Reply:
Also I see a new conflict in api/event-source.md but not sure how to resolve, shall I accept the upstream entirely?